concent_api/concent_api/settings/development.py

```diff
@@ -0,0 +1,7 @@
+from .base import *
+
+SECRET_KEY = 'testkey'
+
+DEBUG = True
+
+ALLOWED_HOSTS = ['localhost']
```

Comment:
You should configure the database here so that developing locally does not require setting it explicitly every time.

``` python
DATABASES['default']['USER']     = 'postgres'
DATABASES['default']['PASSWORD'] = ''
```

Reply:
Okey, fixed!